docs/features/basic-concepts/fee.md

```diff
@@ -31,4 +31,4 @@ Example
 ```
     iriscli stake unbond  --from=test --shares-amount=10 --address-validator=faa1mahw6ymzvt2q3lu4pjj5pau2e8krntklgarrxy  --fee=0.12iris --gas=20000 --chain-id=<chain-id>
 ```
-This example is a transaction to complete the unbond operation. The maximum fee(--fee) is set to be 0.12iris and the maximum(--gas) gas is set to be 20000. Therefore, the gas price here is 0.12^18iris-atto/Gas. Suppose that 10000 gas is used to execute the transaction, then 1200iris-atto will be paid to validators and lefted 0.11999iris will be returned to user.
+This example is a transaction to complete the unbond operation. The maximum fee(--fee) is set to be 0.12iris and the maximum(--gas) gas is set to be 20000. Therefore, the gas price here is 6000iris-nano/Gas. Suppose that 10000 gas is used to execute the transaction, then 1200iris-atto will be paid to validators and lefted 0.11999iris will be returned to user.
```

Comment:
6000iris-nano/gas * 10000gas = 6\*10^7 iris-nano = 0.06iris = 6\*10^16 iris-atto; 0.06iris will be returned to user.